docs/stories/stories.md

```diff
@@ -0,0 +1,79 @@
+# COGS User Stories
+
+## Roles:
+User
+Project owner
```

Comment:
The idea here was that "project owner" (to our thinking), is the person submitting the project form. Often in the current system, at least at Swansea, this could be a different person from the technical lead. I agree that the technical lead is also something worth keeping track of, but not convinced that we should restrict project ownership to technical leads only. Hence decoupling the two.

Reply:
A good question. I'll ask this tomorrow.